demo/atom/spinner/playground

```diff
@@ -0,0 +1,79 @@
+const boxStyle = {
+  backgroundColor: '#ffffff',
+  border: '1px dashed lightgrey',
+  borderRadius: '5px',
+  padding: '10px'
+}
+
+const buttonStyle = {
+  backgroundColor: '#ffffff',
+  padding: '10px'
+}
+
+class SpinnerWrapperExample extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {contentLoaded: false}
+  }
+
+  componentDidMount () {
+    setTimeout(() => {
+      this.setState({contentLoaded: true})
+    }, 5000)
+  }
+
+  render () {
+    const {contentLoaded} = this.state
+    return contentLoaded
+      ? <p style={boxStyle}>Content loaded, spinner dissapears</p>
+      : (
+        <AtomSpinner on={!this.state.contentLoaded} delayed>
```

Comment:
Why the "on" property instead of simple removing the spinner ??

Reply:
forgot to remove the prop as not needed